.github/workflows/watch-dependencies.yaml

```
@@ -160,7 +161,8 @@ jobs:
           releases = data["releases"]
           latest_version = sorted(releases.keys(), key=packaging.version.Version)[-1]
 
-          print(f"::set-output name=version::{latest_version}")
+          with open(os.environ["GITHUB_OUTPUT"], "a") as f:
+              f.write(f"version={latest_version}")
```

```suggestion
              f.write(f"version={latest_version}\n")
```
I don't think `f.write` includes a newline

---

Hmmm hmmm, but we are in append mode (`a`), so I would presume it leads with a new line. Testing a bit.

---

I was considering if `echo hi >> myfile.txt` would cause a new line to be appended or prepended etc, but appending is the way to go I've confirmed now rather than thinking about prepending.